src/components/Text.tsx

```diff
@@ -84,21 +84,32 @@ export function LargeStrokeText({
   )
 }
 
-const bodyText = (small?: boolean, bold?: boolean, center?: boolean) =>
+const bodyText = (
+  small?: boolean,
+  large?: boolean,
+  bold?: boolean,
+  center?: boolean
+) =>
   classnames(
     textColor('text-formal-accent'),
     lineHeight(small ? 'leading-5' : 'leading-6'),
-    fontSize(small ? 'text-sm' : 'text-base'),
+    fontSize(small ? 'text-sm' : large ? 'text-lg' : 'text-base'),
```

Comment:
Maybe it's better to make size instead of small and large?

Reply:
Or separate component into two independent pieces.